packages/core/src/controllers/LimitController.ts

```diff
@@ -0,0 +1,35 @@
+import { subscribeKey as subKey } from 'valtio/vanilla/utils'
+import { proxy } from 'valtio/vanilla'
+
+// -- Types --------------------------------------------- //
+export interface LimitControllerState {
+  pendingTransactions: number
+}
+
```

Comment:
Is this only used in wagmi? if so, can we move to its corresponding adapter folder to prevent increasing base appkit bundle size ?

I see why you called it a controller (valtio store and subscriptions) but I do think this is more of a util instead